File: milestone 2/python_modules/module00/ex6/ft_garden_summary.cpp
#include <iostream>
#include <string>

void	ft_garden_summary(void)
{
	int	plant_num;

	std::string garden_name;
	std::cout << "Enter garden name: ";
	std::getline(std::cin, garden_name);
	std::cout << "Enter number of plants: ";
	std::cin >> plant_num;
	std::cout << garden_name << " has " << plant_num << " plants growing well!\n";
}

int	main(void)
{
	ft_garden_summary();
	return (0);
}

File: milestone 2/python_modules/module00/ex6/ft_garden_summary.java
import java.util.Scanner;

public class ft_garden_summary {
    public static void main(String[] args) {
        System.out.print("Enter garden name: ");
        Scanner scan = new Scanner(System.in);
        String gardenName = scan.nextLine();
        System.out.print("Enter number of plants: ");
        Scanner scan2 = new Scanner(System.in);
        int plant_num = scan2.nextInt();
        System.out.println(gardenName + " has " + plant_num + " plants growing well!");
    }
}